src/components/templates/index.js

```diff
@@ -1,2 +1,4 @@
+export * from './movieList';
```

Comment:
import all, then export through object as mentioned in the comments above

Reply:
closing this PR considering the time constraints, will cover this point in future branches